src/metrics.rs

```diff
@@ -510,6 +512,26 @@ pub fn rpc_metrics() -> &'static RpcMetrics {
             &["type"]
         )
         .unwrap(),
+        serialization_time: register_histogram_vec!(
+            "serialization_time",
+            "Time to perform response data serialization (millis)",
+            &["type", "worker_id"],
+            vec![
+                0.0, 1.0, 5.0, 10.0, 30.0, 50.0, 70.0, 100.0, 300.0, 500.0, 800.0, 1000.0, 5000.0,
+                10_000.0,
+            ]
+        )
+        .unwrap(),
+        account_data_len: register_histogram_vec!(
+            "account_data_len",
+            "Length of account data in bytes, before serialization",
+            &["worker_id"],
+            vec![
+                0.0, 128., 512., 1024., 2048., 4096., 8192.0, 16384., 32768., 65536., 262144.,
```

Comment:
I expect larger values to be more interesting, so I suggest less points in the lower range and more in the higher range like 1Mb, 5Mb, 10Mb, 20Mb, 50Mb